examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -193,6 +194,10 @@ def run_benchmark_variant(
     # Create the actual command for the variant
     variant_command = formulate_benchmark_command(benchmark_dict, variant_dict, args)
 
+    # Check if benchmark calls poprun and might require some env variables,
+    # and modify if needs be
+    variant_command = check_poprun_env_variables(benchmark_name, variant_command)
```

Comment:
Does this need to be tied to Poprun? Could all commands be checked for environment variables that are requested? It feels like an arbitrary limitation even if it is the main use case.

Reply:
I agree with the idea but I think it is out of the scope for this task. I would like to keep the scope of this PR to poprun, and then I can open a new PR to generalise this check to all env vars. I would like to avoid scope creep really.